webviews/sessionLogView/sessionView.tsx

```diff
@@ -8,38 +8,52 @@ import type monacoType from 'monaco-editor';
 import * as monaco from 'monaco-editor/esm/vs/editor/editor.main';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
+import type { SessionPullInfo } from '../../src/common/timelineEvent';
 import { CodeView } from './codeView';
 import './index.css'; // Create this file for styling
 import { parseDiff, type SessionInfo, type SessionResponseLogChunk } from './sessionsApi';
-
+import { vscode } from '../common/message';
 
 interface SessionViewProps {
+	readonly pullInfo: SessionPullInfo | undefined;
 	readonly info: SessionInfo;
 	readonly logs: readonly SessionResponseLogChunk[];
 }
 
 export const SessionView: React.FC<SessionViewProps> = (props) => {
 	return (
 		<div className="session-container">
-			<SessionHeader info={props.info} />
+			<SessionHeader info={props.info} pullInfo={props.pullInfo} />
 			<SessionLog logs={props.logs} />
 		</div>
 	);
 };
 
 // Session Header component
 interface SessionHeaderProps {
+	pullInfo: SessionPullInfo | undefined;
 	info: SessionInfo;
 }
 
-const SessionHeader: React.FC<SessionHeaderProps> = ({ info }) => {
+const SessionHeader: React.FC<SessionHeaderProps> = ({ info, pullInfo }) => {
 	const createdAt = new Date(info.created_at);
 	const completedAt = info.completed_at ? new Date(info.completed_at) : new Date();
 	const durationMs = completedAt.getTime() - createdAt.getTime();
 	const durationSec = Math.round(durationMs / 1000);
 
 	return (
 		<header className="session-header">
+			{pullInfo && (
+				<button
+					className="session-pull-button"
+					onClick={() => {
+						vscode.postMessage({ type: 'openPullRequestView' });
+					}}>
```

Comment:
You could also just use a link and use the PR URI format. 

https://github.com/microsoft/vscode-pull-request-github/blob/1e2ed4ef127421ed04ec8a0b50decd26d0385184/src/common/uri.ts#L543-L546